### PR TITLE
continue adding --ebpf option to the python tools/ scripts

### PR DESCRIPTION
**Pull request by natoscott**
Several python tools allow their eBPF code to be printed to
stdout for debugging.  There are other projects that would
like to share these program definitions however, instead of
duplicating code.  We previously agreed on an --ebpf option
and we now continue adding it to more tools.

Signed-off-by: Nathan Scott <nathans@redhat.com>

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: tools/execsnoop.py [hunk 1]
```diff
@@ -128,8 +130,13 @@
 }
 """
 
+bpf_text.replace("MAXARG", args.max_args)
```
Comment (yonghong-song) [inline tools/execsnoop.py]: You need `bpf_text = bpf_text.replace(...)`.  Otherwise, the statement has no effect and the tool will fail.

**Comment by brendangregg** [comment]
other than @yonghong-song 's comment, LGTM.

**Comment by natoscott** [comment]
@yonghong-song good catch - I've updated the PR with that fix.

**Comment by yonghong-song** [comment]
Looks good. Thanks.

**Pull request merged**